website/docs/integration/driver/jdbc.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Kylin JDBC Driver
+language: en
+sidebar_label: Kylin JDBC Driver
+pagination_label: Kylin JDBC Driver
+toc_min_heading_level: 2
+toc_max_heading_level: 6
+pagination_prev: null
+pagination_next: null
+keywords:
+    - JDBC
+draft: false
+last_update:
+    date: 11/21/2022
+---
+
+Kylin provides JDBC driver, which enables BI or other applications with JDBC interface to access Kylin instance.  
+
+### How to get JDBC Driver
+
+There are two methods to get JDBC Driver:
+
+1. **(Recommended)** You can get Kylin JDBC driver by executing the scripts from `${KYLIN CODES HOME}/build/release/jdbc_package.sh` to generate a full JDBC driver tar package and unzip it to get driver.
+
+2. You can get Kylin JDBC driver from Kylin's installation directory's subdirectory `./lib` , and placed in the BI or other third party applications specified path.
```

Comment:
I think this is a recommended way for most user.

Reply:
fixed